.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
are these vscode files supposed to be committed?

Reply:
They should be. There is nothing here that is user specific.